omenai-mobile/services/orders/getOrdersForUser.tsx

```diff
@@ -0,0 +1,38 @@
+import { OrderTabsTypes } from "screens/orders/Orders";
+import { apiUrl } from "../../constants/apiUrl.constants";
+import { getAsyncData } from "utils/asyncStorage.utils";
+
+export async function getOrdersForUser(orderType: OrderTabsTypes){
+
+    let userId = ''
```

Comment:
Create a guard that prevents accessing the backend unless there is an active session id